docs/usage/getting-started/running.md

```diff
@@ -18,6 +18,26 @@ Self-hosting Renovate means that you are the "administrator" of the bot, which e
 
 ### Available distributions
 
+#### Forking Renovate app
+
+"Forking Renovate" is the sister app to Renovate Bot.
+The difference is that Forking Renovate operates using forks of repositories instead of as a collaborator, so needs only read-only permissions rather than read-write.
```

Comment:
"operates using forks of repositories instead of as a collaborator" this is a really weird sentence, do you have a better wording?

Probably a good idea to change the wording on the Forking Renovate app landing page as well, because that's the source of this text. 😉